src/Microsoft.PowerShell.Commands.Utility/commands/utility/CSVCommands.cs

```diff
@@ -512,7 +512,7 @@ public sealed
         /// <summary>
         /// Mandatory file name to read from.
         /// </summary>
-        [Parameter(Position = 0, ValueFromPipeline = true)]
+        [Parameter(Position = 0, ValueFromPipelineByPropertyName = true)]
```

Comment:
For consistency with the other cmdlets, it is `LiteralPath` that should be bound, not `Path`.

Reply:
I used `Get-Content` as sample. Can you point better sample?

Reply:
Search for instances of `[Alias("PSPath")]` in the source code. If `Get-Content` behaves differently, that's a bug that should be fixed too.

Reply:
Do you suggest remove `ValueFromPipelineByPropertyName` from `Path` parameter?
I found the same in `WSMAnInstance` (Set-WSManInstance).
I think it is not related to the PR and we need to review this in depth in an issue if it is important.

Reply:
@mklement0 Thanks! I removed the `ValueFromPipelineByPropertyName` from `Path` parameter.